manifests/metallb/metallb-openstack-service-lb.yml

```diff
@@ -30,8 +31,9 @@ metadata:
   namespace: metallb-system
 spec:
   addresses:
-    - 10.234.0.0/24
+    - 10.234.0.128/25
```

Comment:
same as above, no need to change the example file unless there's a specific reason for it.